scripts/UPDATE.sh

```diff
@@ -231,6 +231,16 @@ fi
 # copy scripts into the path
 cp -vp $SIRF_SRC_PATH/CCPPETMR_VM/scripts/update*sh $SIRF_INSTALL_PATH/bin
 
+# install the SIRF-Exercises
+cd $SIRF_SRC_PATH
+clone_or_pull SIRF-Exercises
+# install ipython notebook and firefox
+#$SIRF_SRC_PATH/CCPPETMR_VM/scripts/update_VM_with_exercises.sh
+echo "#! /bin/bash
```

Comment:
these ones cannot be here.
something like this should be in the SIRF-exercises/scripts repo.
Presumably we need different content for the jupyter notebook. (if you like, could have 2 scripts, but really that's overkill for now).

Reply:
What cannot be here? The `start_notebook`?

Reply:
On Slack @KrisThielemans  and I have agreed that we will add info in the SIRF-Exercises repository on how to start the jupyter notebook.